<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="" xml:lang="">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Jasmine Gupta1, Chelsey Fang1, Thomas Devlin1 1Space Technologies at Cal, UC Berkeley, Berkeley, CA USA" />
  <title>Ken Souza Memorial Spaceflight Research Program 2020 Proposal - STAC</title>
  <style>
    html {
      color: #1a1a1a;
      background-color: #fdfdfd;
    }
    body {
      margin: 0 auto;
      max-width: 36em;
      padding-left: 50px;
      padding-right: 50px;
      padding-top: 50px;
      padding-bottom: 50px;
      hyphens: auto;
      overflow-wrap: break-word;
      text-rendering: optimizeLegibility;
      font-kerning: normal;
    }
    @media (max-width: 600px) {
      body {
        font-size: 0.9em;
        padding: 12px;
      }
      h1 {
        font-size: 1.8em;
      }
    }
    @media print {
      html {
        background-color: white;
      }
      body {
        background-color: transparent;
        color: black;
        font-size: 12pt;
      }
      p, h2, h3 {
        orphans: 3;
        widows: 3;
      }
      h2, h3, h4 {
        page-break-after: avoid;
      }
    }
    p {
      margin: 1em 0;
    }
    a {
      color: #1a1a1a;
    }
    a:visited {
      color: #1a1a1a;
    }
    img {
      max-width: 100%;
    }
    svg {
      height: auto;
      max-width: 100%;
    }
    h1, h2, h3, h4, h5, h6 {
      margin-top: 1.4em;
    }
    h5, h6 {
      font-size: 1em;
      font-style: italic;
    }
    h6 {
      font-weight: normal;
    }
    ol, ul {
      padding-left: 1.7em;
      margin-top: 1em;
    }
    li > ol, li > ul {
      margin-top: 0;
    }
    blockquote {
      margin: 1em 0 1em 1.7em;
      padding-left: 1em;
      border-left: 2px solid #e6e6e6;
      color: #606060;
    }
    code {
      font-family: Menlo, Monaco, Consolas, 'Lucida Console', monospace;
      font-size: 85%;
      margin: 0;
      hyphens: manual;
    }
    pre {
      margin: 1em 0;
      overflow: auto;
    }
    pre code {
      padding: 0;
      overflow: visible;
      overflow-wrap: normal;
    }
    .sourceCode {
     background-color: transparent;
     overflow: visible;
    }
    hr {
      border: none;
      border-top: 1px solid #1a1a1a;
      height: 1px;
      margin: 1em 0;
    }
    table {
      margin: 1em 0;
      border-collapse: collapse;
      width: 100%;
      overflow-x: auto;
      display: block;
      font-variant-numeric: lining-nums tabular-nums;
    }
    table caption {
      margin-bottom: 0.75em;
    }
    tbody {
      margin-top: 0.5em;
      border-top: 1px solid #1a1a1a;
      border-bottom: 1px solid #1a1a1a;
    }
    th {
      border-top: 1px solid #1a1a1a;
      padding: 0.25em 0.5em 0.25em 0.5em;
    }
    td {
      padding: 0.125em 0.5em 0.25em 0.5em;
    }
    header {
      margin-bottom: 4em;
      text-align: center;
    }
    #TOC li {
      list-style: none;
    }
    #TOC ul {
      padding-left: 1.3em;
    }
    #TOC > ul {
      padding-left: 0;
    }
    #TOC a:not(:hover) {
      text-decoration: none;
    }
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    /* The extra [class] is a hack that increases specificity enough to
       override a similar rule in reveal.js */
    ul.task-list[class]{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      font-size: inherit;
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    .display.math{display: block; text-align: center; margin: 0.5rem auto;}
  </style>
</head>
<body>
<header id="title-block-header">
<h1 class="title">Ken Souza Memorial Spaceflight Research Program 2020
Proposal - STAC</h1>
<p class="author">Jasmine Gupta<sup>1</sup>, Chelsey Fang<sup>1</sup>,
Thomas Devlin<sup>1</sup><br />
<sup>1</sup>Space Technologies at Cal, UC Berkeley, Berkeley, CA USA</p>
<p class="date">June 2020</p>
</header>
<h1 id="introduction">Introduction</h1>
<p>The increasing prevalence of antibiotic resistance among pathogenic
bacteria is a problem that threatens global human health. In 2013, the
CDC declared that the human race is in the “post-antibiotic era” and the
World Health Organization has also warned that the crisis has become
dire.<sup>1</sup> The CDC estimates that it costs the American
healthcare system 20 billion dollars per year in direct costs, with
indirect lost productivity costs of up to 35 billion
dollars.<sup>2</sup> Consequently, understanding how microbes obtain
resistance to antibiotics will help us reverse-engineer solutions to the
problem.</p>
<p>The challenging field of human space flight contends with many
physiological and biological challenges. Some are well documented, such
as muscle atrophy due to lack of gravity, others, like the decrease in
antibiotic efficacy are less well understood.<sup>3,4</sup>
Microorganisms have the ability to respond almost immediately to
environmental changes.<sup>5</sup> This ability allows them to adapt and
survive in extreme environments, such as microgravity. Literature has
discussed the upregulation of genes involved in “starvation response,
acid stress, osmotic stress, oxidative stress, biofilm formation, curli
biosynthesis and lipid biosynthesis” upon exposure to
microgravity<sup>5</sup>. Because the genes for biofilm formation are
upregulated, there is greater biofilm production, creating thicker and
more robust biofilms.<sup>5</sup> These biofilms prevent antibiotics
from working effectively or even at all. As such, there is higher
resistance to antibiotics in microgravity. The phenomena of increased
resistance has been shown in a previous study aboard the ISS using both
E. coli and the antibiotic Gentamicin.<sup>6</sup></p>
<p>We aim to utilize the unique opportunity presented by Blue Origin’s
New Shepard flight to test the effects of short term exposure to
microgravity in tandem with long-term simulated microgravity experiments
on the ground to understand how exposure to microgravity affects
bacterial resistance to antibiotics. We will expand upon previous
studies and focus on how changes in gene expression correspond to
changes in resistance.</p>
<h1 id="hypothesis">Hypothesis</h1>
<p>The combined stress of launch into spaceflight and subsequent
microgravity environment activates a change in bacterial gene expression
that renders them less susceptible to the antibiotic Gentamicin. Further
exploration of differential gene expression will elucidate general
mechanisms that result in antibiotic resistance.</p>
<h1 id="research-objectives">Research Objectives</h1>
<ol>
<li><p>To utilize microgravity to understand the development of
evolutionary resistance to antibiotics.</p></li>
<li><p>To understand the effects of microgravity on the clinical
microbiome as well as on bacterial transcriptomics and
physiology.</p></li>
<li><p>To demonstrate the utility of commercial spaceflight for
proof-of-concept biological experiments.</p></li>
</ol>
<h1 id="experimental-design">Experimental Design</h1>
<p>When bacteria is in need of a change in protein composition due to an
outside stress, RNA polymerases are recruited to create RNA templates of
the corresponding sections of the DNA genome. These templates are made
into the desired proteins by ribosomes. This ability to control protein
production allows cells to adapt and survive in extreme environments
such as microgravity. By reading the RNA that is present in a culture
sample, one can see which genes are in greater or lesser demand, and
therefore one can tell how the cell is adapting to environmental
changes.</p>
<figure id="fig:my_label">
<img src="images/sequence1.png" />
<figcaption>Basic Experiment Workflow</figcaption>
</figure>
<p>The general design of our microgravity experiment consists of five
parts (see Fig 1):</p>
<ol>
<li><p>Before delivery of the payload, we will inoculate E. coli
bacteria into LB media at a concentration that will ensure the bacteria
will be in the log phase of growth during flight. We plan to determine
this concentration through experiments to characterize the growth curve
of E. coli at the maintained temperature of our payload. This ensures
the maximum amount of change in gene expression and therefore maximizes
the data significance. Addressing concerns regarding the inflight
culture of E. coli, it is a low-maintenance bacteria and easily
replicates once provided solely with media, therefore, once in LB, it
will continue to grow until saturation or RNAlater (RNA fixative)
injection in our case. We plan to use the Pad Load option to improve the
viability of our colony.</p></li>
<li><p>We will expose the variable plates of E. coli to the antibiotic
Gentamicin at various time points, shown in Fig 2.<br />
</p>
<figure id="fig:my_label">
<img src="images/timeline2.png" />
<figcaption>Timeline for Injection of Antibiotic and RNAlater during
flight</figcaption>
</figure></li>
<li><p>We will arrest bacterial growth and preserve the RNA expression
profile of the bacteria through fixation with RNALater (see Fig 2).
RNAlater is a fixative which “freezes” gene expression by stabilizing
RNA transcripts in a solution. We have successfully made RNALater in the
lab and have used it to preserve extracted RNA for 24 hours at room
temperature and long term at -20<sup>o</sup>C. This allows us to analyze
the gene expression of E. coli in microgravity after the payload has
returned to earth. Our payload will be delivered to our group on dry ice
(-80<sup>o</sup>C) within 24 hours of return for analysis.</p></li>
<li><p>Concurrently, the same procedure will be executed on the ground
in order to differentiate between RNA composition changes that occur
normally, versus those that occur as a result of our experiment. We will
perform an RNA extraction on both the ground sample and payload, and
will send our results for sequencing to the Berkeley Quantitative
Biosciences genomic sequencing facility. We have honed our procedure for
RNA extraction through multiple experiments and have consistently
obtained quality RNA. We have checked its quality through gel
electrophoresis and have not seen any degradation or
contamination.</p></li>
<li><p>We will examine our RNA sequencing results to observe the
differences in gene expression between the bacteria from the payload and
the bacteria from the control experiment on the ground.</p></li>
</ol>
<p>As shown in Fig 2, we will have 5 plates in the payload, allowing us
to implement different conditions with each plate containing 5
replicates. For our first plate, we aim to have a control to see the
overall differential gene expression over the entire course of the
flight. Plate 2 fixes the cells right after we exit microgravity
conditions while Plate 3 fixes them right before entering microgravity.
With these 3 controls we are able to isolate the changes in gene
expression at each portion of the flight and the specific responses to
solely hypergravity or microgravity by comparing them with results from
the ground experiment. This will allow us to determine what changes are
specifically due to the added stress of the antibiotic in each specific
condition. Plates 4 and 5 are our test conditions, with the antibiotic
injected at the beginning of the condition in question and RNAlater
injected at the end of said condition.</p>
<h1 id="engineering">Engineering</h1>
<p>The main engineering challenge of our project focuses on the design
and automation of a flight-ready payload for parabolic launch on the New
Shepard Rocket. At the time of our proposal last year, our payload’s
flight hardware was in early stages of development. Feedback we received
after the proposal noted that, “the flight hardware design appears
prohibitively difficult to achieve in a 2U payload format”. Since the
time of that proposal we have developed and prototyped that hardware
extensively and made vast improvements on our old designs.</p>
<figure id="fig:my_label">
<img src="images/plate.png" />
<figcaption>A single treatment plate</figcaption>
</figure>
<p>One concern was the weight limit. Our current mass budget takes all
systems and components into account and comes in at 478 g, which is
under the 500 g limit. This budget is based not only off of nominal
masses found in documentation, but also empirically verified assembly
masses. Another issue was the difficulty of implementing a precise
injection system. As our experiment requires injection of two fluids,
our solution has two parts. For RNALater to be effective, we inject a
1:1 ratio of the reagent to E. Coli medium. This means that we can err
on the side of injecting extra reagent, which we do. In the case of the
antibiotic, precision is necessary. For this reason we have iterated
over several injection methods. Our current method for antibiotic
injection, which takes advantage of precise micropipetting, has been
tested and has less than 2.5% error. Because consistency is a key
factor, we have developed and tested loading procedures for both
fluids.</p>
<p>We also have fully developed electronics and software ready for
testing. We have custom printed circuit boards which will soon have
components mounted for testing. We also have software written to run on
an Arduino processor and have conducted preliminary tests using a flight
simulator. As noted in the feedback we received, the growth stage of the
<em>E. coli</em> at launch is very important. A temperature regulation
system will maintain a constant temperature within an acceptable range
for growth. Given a constant temperature, we can create a growth curve
to determine the cell density of E. coli necessary to achieve log phase
growth during microgravity and dilute our inoculation accordingly. This
timing ensures that cells will be growing exponentially in microgravity,
providing maximum RNA expression. Including this heating system, we have
created a power budget totalling 2.38 W, and expect to be able to run
off of the USB power supplied by Blue Origin.</p>
<figure id="fig:my_label">
<img src="images/Chassis.JPG" />
<figcaption>Overall chassis assembly</figcaption>
</figure>
<p>Under the parameters set by Blue Origin, we have constructed a
chassis whose dimensions lie within the 2U requirement. Additionally,
due to the biological aspect of this experiment, two levels of
containment are required. The first layer of containment is a layer of
BreatheEasy, a gas-permeable membrane to seal the plate and restrict
liquid flow. For the second layer of containment, there are removable
sheets that will slide and lock into each side of the chassis during the
experiment and have the ability to release after the experiment in order
to collect the samples. Considering the experimental plates need to be
easily accessible before and after the experiment for analysis, we have
designed the chassis to structurally support each plate and allow for
easy access, yet strong connections. Furthermore, we have designed the
chassis with carbon fiber supports to give us high stiffness and low
weight to help accommodate the 500 g weight limit.</p>
<h1 id="public-engagement">Public Engagement</h1>
<figure id="fig:my_label">
<img src="images/outreach.jpg" />
<figcaption>Outreach Efforts</figcaption>
</figure>
<p>Another focus of our team is to increase the accessibility of space
and inspire STEM education in the East Bay community. This team is a
part of Space Technologies at Cal (STAC), a group of 60 passionate UC
Berkeley students who are working on a range of technologies aimed at
making strides in space research. We continually engage our peers at UC
Berkeley and the community at large, building on our past experience and
applying it to the public engagement aspect of this payload. For the
past few years, we have invited high school students to collaborate with
us during summers to get hands on experience designing and building
technology for space applications. We have plans to further connect with
local high schools to talk about current space news as well as
collaborate on educational projects such as HAB launches. The past two
years, STAC has hosted a Space Tech Symposium, inviting industry
members, academia, and VC’s in the Bay Area to speak on panels and
create an event that brings people interested in Space together. A
successful launch would help spur interest in the event and help inspire
other groups to learn and experiment in Space. We also plan to continue
submitting our finalized research results to a variety of conferences
(like ASGSR, ASM, etc.), writing publically available blog posts on
Medium, and promoting external education student outreach.</p>
<h1 id="conclusion">Conclusion</h1>
<p>The opportunity to conduct cheaper, more inexpensive microgravity
experiments with Blue Origin represents a paradigm shift in how we can
use spaceflight to tackle conventional biological questions. The
proposed project demonstrates the ability for a student-led research
initiative (STAC) to tackle a pressing biomedical problem: the issue of
global antibiotic resistance.</p>
<h1 id="current-relationships-and-collaborators">Current Relationships
and Collaborators</h1>
<ul>
<li><p>Steve Ruzin: UC Berkeley Faculty Mentor, Director of the CNR
Biological Imaging Facility</p></li>
<li><p>Gian Garriga: UC Berkeley Professor of Molecular and Cell
Biology</p></li>
<li><p>Antonio Ricco: Chief Technologist at NASA Ames for Small
Payloads</p></li>
<li><p>Luis Zea: Assistant Research Professor at University of Colorado,
Boulder with Bioserve Technologies</p></li>
</ul>
<h1 id="references-links">References (links)</h1>
<ol>
<li><p><span style="color: blue"><a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4165128/">www.ncbi.nlm.nih.gov
(1)</a></span></p></li>
<li><p><span style="color: blue"><a
href="https://www.cdc.gov/media/releases/2013/p0916-untreatable.html">www.cdc.gov</a></span></p></li>
<li><p><span style="color: blue"><a
href="https://www.nasa.gov/pdf/64249main_ffs_factsheets_hbp_atrophy.pdf">www.nasa.gov</a></span></p></li>
<li><p><span style="color: blue"><a
href="https://www.sciencedirect.com/science/article/abs/pii/S0167779906000266">www.sciencedirect.com</a></span></p></li>
<li><p><span style="color: blue"><a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3589462/">www.ncbi.nlm.hib.gov
(2)</a></span></p></li>
<li><p><span style="color: blue"><a
href="https://www.colorado.edu/faculty/zea-luis/sites/default/files/attached-files/zea_-_thesis_-_published.pdf">www.colorado.edu/faculty/zea-luis</a></span></p></li>
</ol>
</body>
</html>
